apps/daimo-mobile/src/action/useSendAsync.ts

```diff
@@ -50,17 +49,14 @@ export function useSendAsync({
 
     // Add pending op and named accounts to history
     if (pendingOp) {
-      pendingOp.opHash = handle.userOpHash as Hex;
+      pendingOp.opHash = handle as Hex;
       pendingOp.timestamp = Math.floor(Date.now() / 1e3);
       account.recentTransfers.push(pendingOp);
       account.namedAccounts.push(...(namedAccounts || []));
       // TODO: add pending device add/removes
       console.log(`[SEND] added pending op ${pendingOp.opHash}`);
 
       setAccount(account);
-
-      // In the background, wait for the op to finish
-      handle.wait().then(() => resync(`op finished: ${pendingOp.opHash}`));
```

Comment:
just to double check @dcposch this `resync` isn't really necessary right? afaict since we have a pending op it resyncs at high frequency anyway?

Reply:
ya, not necessary, feel free to drop